test/kubemark/iks/util.sh

```diff
@@ -163,24 +167,26 @@ function complete-login {
 function generate-values {
 	echo "Generating values"
 	master-config
+	# shellcheck disable=SC2034 # KUBECTL used in parent script
 	KUBECTL=kubectl
 	KUBEMARK_DIRECTORY="${KUBE_ROOT}/test/kubemark"
 	RESOURCE_DIRECTORY="${KUBEMARK_DIRECTORY}/resources"
+	# shellcheck disable=SC2034 # TEST_CLUSTER_API_CONTENT_TYPE used in parent script
 	TEST_CLUSTER_API_CONTENT_TYPE="bluemix" #Determine correct usage of this
 	CONFIGPATH=${KUBECONFIG%/*}
-	KUBELET_CERT_BASE64="${KUBELET_CERT_BASE64:-$(cat ${CONFIGPATH}/admin.pem | base64 | tr -d '\r\n')}"
-	KUBELET_KEY_BASE64="${KUBELET_KEY_BASE64:-$(cat ${CONFIGPATH}/admin-key.pem  | base64 | tr -d '\r\n')}"
-	CA_CERT_BASE64="${CA_CERT_BASE64:-$(cat `find ${CONFIGPATH} -name *ca*` | base64 | tr -d '\r\n')}"
+	KUBELET_CERT_BASE64="${KUBELET_CERT_BASE64:-$(base64 "${CONFIGPATH}/admin.pem" | tr -d '\r\n')}"
+	KUBELET_KEY_BASE64="${KUBELET_KEY_BASE64:-$(base64 "${CONFIGPATH}/admin-key.pem" | tr -d '\r\n')}"
+	CA_CERT_BASE64="${CA_CERT_BASE64:-$( base64 "$(find "${CONFIGPATH}" -name "*ca*" | head -n 1)" | tr -d '\r\n')}"
```

Comment:
`head -n 1`?

Reply:
From what I had originally gathered, `CA_CERT_BASE64` and eventual generated kubeconfig was expecting a single ca pem file to be read and encoded. The large glob used (`*ca*`) could potentially pick up multiple results. The `head -n 1` was a means to limit it to always return a single result. However there is potential for it to return the wrong ca or match other files. In either case a failure would occur, but the limit at least ensures its always getting a single result.